core/common/src/main/java/alluxio/conf/InstancedConfiguration.java

```diff
@@ -70,7 +70,7 @@ public class InstancedConfiguration implements AlluxioConfiguration {
    * @return an instanced configuration preset with defaults
    */
   public static InstancedConfiguration defaults() {
-    return new InstancedConfiguration(ConfigurationUtils.defaults());
+    return new InstancedConfiguration(ConfigurationUtils.copyDefaults());
```

Comment:
copyDefaults here?

Reply:
No, `InstancedConfiguration` has `set` method, so it is not safe to just change this to return `defaults()`.